russian/prayer/exmachina/exmachina.md

```diff
@@ -0,0 +1,19 @@
+"Не тарантино. конечно. но делать нечего. Тем более, что я забыл напрочь, куда и зачем шел." - подумал ты.
+
+Решено. Ты бредешь в кинозал. Весь такой из себя махровый дед бредешь на совсем не олдскульный кинч.
+
+Тебе бы на что-то попроще сходить, но  нет... Ты шаришь в ИТ - винду от Паскаля отличить умеешь. Значит ты в теме.
+
+Тем более ты помнишь, как облажался на последних трансформерах, когда не выдержал спецеффектов и повалился с выпученными глазами еще до входа в кинозал.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.